### PR TITLE
Provide ZoneId in CSM

### PR DESCRIPTION
**Pull request by TurpIF**
JIRA: EE-1124

### CODE REVIEW

File: publisher-sdk/src/main/java/com/criteo/publisher/csm/Metric.java [hunk 1]
```diff
@@ -68,6 +68,14 @@
   @Nullable
   abstract String getRequestGroupId();
 
+  /**
+   * Zone ID that was mapped during bidding to the CDB slot response.
+   * <p>
+   * This should be present in case of valid bid.
```
Comment (Amokrane) [inline publisher-sdk/src/main/java/com/criteo/publisher/csm/Metric.java:74]: should or must? https://www.ietf.org/rfc/rfc2119.txt

File: publisher-sdk/src/main/java/com/criteo/publisher/model/CdbResponseSlot.kt [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import com.criteo.publisher.DependencyProvider
+import com.criteo.publisher.annotation.OpenForTesting
+import com.criteo.publisher.model.nativeads.NativeAssets
+import com.criteo.publisher.util.URLUtil
+import com.google.gson.annotations.SerializedName
+import org.json.JSONObject
+import java.io.ByteArrayInputStream
+
+@OpenForTesting
+data class CdbResponseSlot(
```
Comment (Amokrane) [inline publisher-sdk/src/main/java/com/criteo/publisher/model/CdbResponseSlot.kt:29]: Why was `Slot` renamed to `CdbResponseSlot` during the data class migration? Is there another `Slot` class?
I prefer `Slot` for simplicity.

File: publisher-sdk/src/main/java/com/criteo/publisher/model/CdbResponseSlot.kt [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    @SerializedName("currency") val currency: String? = null,
+    @SerializedName("width") val width: Int = 0,
+    @SerializedName("height") val height: Int = 0,
+    @SerializedName("displayUrl") val displayUrl: String? = null,
+    @SerializedName("native") val nativeAssets: NativeAssets? = null,
+
+    /**
+     * The TTL in seconds for this bid response.
+     */
+    @SerializedName("ttl") var ttl: Int = 0,
```
Comment (Amokrane) [inline publisher-sdk/src/main/java/com/criteo/publisher/model/CdbResponseSlot.kt:43]: I suggest renaming to `ttlInSeconds`.

File: publisher-sdk/src/main/java/com/criteo/publisher/csm/Metric.java (hunk 1 above)
Comment (TurpIF) [reply publisher-sdk/src/main/java/com/criteo/publisher/csm/Metric.java:74]: Normally it is "must". But we do not control the TLA and this information comes from it.

File: publisher-sdk/src/main/java/com/criteo/publisher/model/CdbResponseSlot.kt (hunk 2 above)
Comment (TurpIF) [reply publisher-sdk/src/main/java/com/criteo/publisher/model/CdbResponseSlot.kt:29]: The is the `CdbRequestSlot` too. This is to make the distinction.

**Pull request merged**